Prog/Fix NL000093 and NL000319.sas

```diff
@@ -0,0 +1,424 @@
+  
+  format _all_ ;
+  informat _all_ ;
+  
+  
+  ** Standard geos **;
+  
+  length Ward2012 $ 1;
+  
+  Ward2012 = substr( Ward_2012, 6, 1 );
+  
+  format Ward2012 $ward12a.;
+  
+  length Anc2012 $ 2;
+  
+  Anc2012 = substr( Anc_2012, 5, 2 );
+  
+  format Anc2012 $anc12a.;
+  
+  length Psa2012 $ 3;
+  
+  Psa2012 = substr( Psa, 21, 3 ); 
+  
+  format Psa2012 $psa12a.;
+  
+  length Geo2010 $ 11;
+  
+  if Census_tract ~= . then Geo2010 = "11001" || put( Census_tract, z6. );
+  
+  format Geo2010 $geo10a.;
+  
+  length Cluster_tr2000 $ 2 Cluster_tr2000_name $ 80;
+  
+  if Cluster_ ~= "" then Cluster_tr2000 = put( 1 * substr( Cluster_, 9, 2 ), z2. );
+  
+  format Cluster_tr2000 $clus00a.;
+  
+  length Zip $ 5;
+  
+  Zip = put( ZIPCODE, z5.0 );
+  
+  format Zip $zipa.;
+
+  ** Cluster names **;
+  
+  Cluster_tr2000_name = put( Cluster_tr2000, $clus00b. );
+  
+  ** Image url **;
+  
+  if imagename ~= "" and imagename ~=: "No_Image_Available" then 
+    bldg_image_url = trim( imageurl ) || "/" || trim( left( imagedir ) ) || "/" || imagename;
+  
+run;
+
+data Building_Geocode;
+set prescat.Building_Geocode;
+run;
+
+data Building_Geocode;
+modify Building_Geocode bldg_mar_address;
+by nlihc_ID;
+run;
+
+
+/*proc sort data=building_geocode;
+ by nlihc_id Bldg_addre;
+ run;*/
+
+title2 '********************************************************************************************';
+title3 '** 3/ Check for changes in building_geocode file unrelated to fixes for NL000093 and NL000319';
+
+proc compare base=prescat.building_geocode compare=building_geocode listall maxprint=(40,32000);
+ id nlihc_id proj_name;
+ run;
+
+
+data proj_mar_address;
+
+  set Mar_address (rename = (address_id = proj_address_id fulladdress=proj_addre Streetviewurl=proj_streetview_url));
+  
+  format _all_ ;
+  informat _all_ ;
+  
+  
+  ** Standard geos **;
+  
+  length Ward2012 $ 1;
+  
+  Ward2012 = substr( Ward_2012, 6, 1 );
+  
+  format Ward2012 $ward12a.;
+  
+  length Anc2012 $ 2;
+  
+  Anc2012 = substr( Anc_2012, 5, 2 );
+  
+  format Anc2012 $anc12a.;
+  
+  length Psa2012 $ 3;
+  
+  Psa2012 = substr( Psa, 21, 3 ); 
+  
+  format Psa2012 $psa12a.;
+  
+  length Geo2010 $ 11;
+  
+  if Census_tract ~= . then Geo2010 = "11001" || put( Census_tract, z6. );
+  
+  format Geo2010 $geo10a.;
+  
+  length Cluster_tr2000 $ 2 Cluster_tr2000_name $ 80;
+  
+  if Cluster_ ~= "" then Cluster_tr2000 = put( 1 * substr( Cluster_, 9, 2 ), z2. );
+  
+  format Cluster_tr2000 $clus00a.;
+  
+  length Zip $ 5;
+  
+  Zip = put( ZIPCODE, z5.0 );
+  
+  format Zip $zipa.;
+
+  ** Cluster names **;
+  
+  Cluster_tr2000_name = put( Cluster_tr2000, $clus00b. );
+  
+  ** Image url **;
+  
+  if imagename ~= "" and imagename ~=: "No_Image_Available" then 
+    proj_image_url = trim( imageurl ) || "/" || trim( left( imagedir ) ) || "/" || imagename;
+  
+run;
```

Comment:
I'm not seeing updates to Proj_lat, Proj_lon, Proj_x, and Proj_y in the Proc Compare output.